en/docs/references/ballerina-expression-syntax.md

```diff
@@ -68,6 +88,50 @@ To initialize a variable of type `var` in Choreo, you can use one of the followi
 "Hello!"
 ```
 
+### error
+
+The `error` type supports error values that include a string message. The expressions to initialize this variable type needs to be entered in the `error("<ERROR_STRING>")` format.
+
+###### Examples:
+
+To initialize a variable of type `error` in Choreo, you can enter an expression as follows:
+
+```ballerina
+error("The item requested is not found.")
+```
+
+### any
+
+The `any` type can represent a variable of any type except the error type. When you do not have prior knowledge of the type of a variable, you can assign it to a variable of type any. Values of these variables can come from dynamic content such as the request and response message references and reference variable types.
```

Comment:
```suggestion
The `any` type represents any value that is not an error. When you do not have prior knowledge of the type of a variable, you can assign it to a variable of type `any`. Values assigned to a variable of type `any` can come from dynamic content such as the request and response message references and reference variable types.
```

Reply:
Do we need to say "When you do not have prior knowledge of the type of a variable, you can assign it to a variable of type `any`"? This isn't quite correct IMO. We will always know the static type.